declarations/Instagram.json

```
@@ -4,7 +4,10 @@
     "Privacy Policy": {
       "fetch": "https://help.instagram.com/519522125107875",
       "select": [
-        "#cms-help-content-results"
+        "#cms-help-content-results > div > div:first-child"
```

I feel that this selector relying on classname over filiation would be more stable:

```suggestion
        "#cms-help-content-results .n"
```